app/views/sufia/admin/admin_sets/_form_visibility.html.erb

```diff
@@ -0,0 +1,20 @@
+          <div id="visibility" class="tab-pane">
+            <div class="panel panel-default labels">
+              <%= simple_form_for @form.permission_template,
+                                  url: [sufia, :admin, @form, :permission_template] do |f| %>
+                <div class="panel-body">
+                  <p><%= t('.page_description') %></p>
+                  <h3><%= t('.visibility.title') %></h3>
+                  <p><%= t('.visibility.description') %></p>
+                  <ul class="visibility">
```

Comment:
Is there a reason we're deviating from the recommended bootstrap markup? http://getbootstrap.com/css/#checkboxes-and-radios